test/language/expressions/async-arrow-function/arrow-returns-promise.js

```diff
@@ -0,0 +1,15 @@
+// Copyright 2016 Microsoft, Inc. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Brian Terlson <brian.terlson@microsoft.com>
+es7id: pending
+description: >
+  Async arrow functions return promises
+---*/
+
+var p = (async () => await 1 + await 2)();
+p.then(function (v) {
+  assert.sameValue(v, 3);
+  $DONE();
+}, $DONE);
```

Comment:
I recently started explicitly invoking `$DONE` with a error description from within a closure, just in case the promise was rejected with out a reason (or with any falsey reason) since [the helper wouldn't report an error in those cases](https://github.com/tc39/test262/blob/fbad34959ffedb02c5b8c040e0f992a6fc006f36/harness/doneprintHandle.js#L5)